### PR TITLE
Add an activation argument to all classifiers

### LINKED ISSUE #954: Add an `activation` argument for all classifiers
For general usability and consistency with keras-cv, we would like to add an `activation` argument to all classifiers.

```python
model = BertClassifier.from_preset(
    "bert_base_en_uncased",
    num_classes=2,
    activation="softmax",
)
model.compile(...)
model.fit(...)
model.predict(["What an amazing movie!", "A total waste of my time."])
>>> np.array([[0.004173, 0.9956  ], [0.997   , 0.0028  ]])
```

One way this could work:
- `BertClassifier` activation defaults to `None` (logits).
- If activation is softmax, we default compile with `loss=keras.losses.SparseCategoricalCrossentropy(from_logits=False)`.
- If activation is `None`, we default compile with `loss=keras.losses.SparseCategoricalCrossentropy(from_logits=True)`.
- If activation is something else, we do not default compile.

### PR DESCRIPTION
**Pull request by mattdangerw**
This will better match recent changes on KerasCV.

Because our language models need to return logits (e.g. to manipulate logits during sampling), we will not update the default behavior of the classifiers for now. We return logits by default across the library, and compile with `from_logits=True` losses across the library.

New quickstart demo -> 
https://gist.github.com/mattdangerw/afc5a7877c9d79b4dfafdf789dc69eae

Fixes #954

### CODE REVIEW

File: keras_nlp/models/albert/albert_classifier.py [hunk 1]
```diff
@@ -47,11 +47,14 @@ class AlbertClassifier(Task):
...
         backbone: A `keras_nlp.models.AlertBackbone` instance.
         num_classes: int. Number of classes to predict.
-        dropout: float. The dropout probability value, applied after the dense
-            layer.
         preprocessor: A `keras_nlp.models.AlbertPreprocessor` or `None`. If
             `None`, this model will not apply preprocessing, and inputs should
             be preprocessed before calling the model.
+        activation: Optional `str` or callable, defaults to `None`. The
+            activation function to use on the model outputs. Set
+            `activation=`"softmax"`` to return output probabilities.
```
Comment (chenmoneygithub) [inline keras_nlp/models/albert/albert_classifier.py:55]: nit: I think we can delete the extra `` - `activation="softmax"`

**Review by chenmoneygithub — APPROVED** [review]
Thanks! LGTM, one minor comment.

File: keras_nlp/models/albert/albert_classifier.py (hunk 1 above)
Comment (mattdangerw) [reply keras_nlp/models/albert/albert_classifier.py:55]: oops, thanks!

**Review by jbischof — APPROVED** [review]
Looks great, thank you!
